FloripaTrans/FTFindRoutesGoogleMapsViewController.m

```diff
@@ -0,0 +1,130 @@
+//
+//  FTFindRoutesGoogleMapsViewController.m
+//  FloripaTrans
+//
+//  Created by Vítor Makoto Matayoshi de Moraes on 3/7/15.
+//  Copyright (c) 2015 Vitor Makoto. All rights reserved.
+//
+
+#import "FTFindRoutesGoogleMapsViewController.h"
+#import <GoogleMaps/GoogleMaps.h>
+#import <CoreLocation/CoreLocation.h>
+#import "UIView+FTViewLayout.h"
+
+@interface  FTFindRoutesGoogleMapsViewController() <GMSMapViewDelegate>
+@property (weak, nonatomic) IBOutlet GMSMapView *mapView;
+@property (strong, nonatomic) GMSMarker *routeMarker;
+
+@property (weak, nonatomic) IBOutlet UIButton *okButton;
+
+@end
+@implementation FTFindRoutesGoogleMapsViewController
+
+- (void)viewDidLoad {
+    
+    
+    self.navigationItem.title = @"Map";
+    
+    // Set up the map view
+    self.mapView.myLocationEnabled = NO;
+    self.mapView.mapType = kGMSTypeNormal;
+    self.mapView.settings.compassButton = YES;
+    self.mapView.settings.myLocationButton = NO;
+    
+    self.mapView.delegate = self;
+    
+    // Round the corner of the search button
+    [self.okButton roundViewCornerWithRadius:10];
+    
+    // Set up initial location for the map view
+    GMSCameraPosition *camera = [GMSCameraPosition cameraWithLatitude:-27.593500
```

Comment:
What is this magic number? We should always use constants for this kind of value.